builtin/providers/aws/config.go

```diff
@@ -3,14 +3,18 @@ package aws
 import (
 	"fmt"
 	"log"
+	"net"
 	"strings"
+	"time"
 
 	"github.com/hashicorp/go-cleanhttp"
 	"github.com/hashicorp/go-multierror"
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/awserr"
-	"github.com/aws/aws-sdk-go/aws/credentials"
+	awsCredentials "github.com/aws/aws-sdk-go/aws/credentials"
```

Comment:
may or may not want to keep this explicitly named, I felt like it was less confusing while developing